_usage/locales.md

```diff
@@ -19,109 +19,120 @@ encoding. Babelfish will automatically translate from the server
+| `bbf_unicode_cp1251_ci_as` | Case insensitive collation for the [Windows-1251 single-byte character encoding](https://www.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WindowsBestFit/bestfit1251.txt). |
+| `bbf_unicode_cp1_ci_as` | Case insensitive collation for the [Windows-1252 single-byte character encoding](https://www.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WindowsBestFit/bestfit1252.txt). |
+| `bbf_unicode_cp1253_ci_as` | Case insensitive collation for the [Windows-1253 single-byte character encoding](https://www.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WindowsBestFit/bestfit1253.txt). |
+| `bbf_unicode_cp1254_ci_as` | Case insensitive collation for the [Windows-1254 single-byte character encoding](https://www.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WindowsBestFit/bestfit1254.txt). |
+| `bbf_unicode_cp1255_ci_as` | Case insensitive collation for the [Windows-1255 single-byte character encoding](https://www.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WindowsBestFit/bestfit1255.txt). |
+| `bbf_unicode_cp1256_ci_as` | Case insensitive collation for the [Windows-1256 single-byte character encoding](https://www.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WindowsBestFit/bestfit1256.txt). |
+| `bbf_unicode_cp1257_ci_as` | Case insensitive collation for the [Windows-1257 single-byte character encoding](https://www.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WindowsBestFit/bestfit1257.txt). |
+| `bbf_unicode_cp1258_ci_as` | Case insensitive collation for the [Windows-1258 single-byte character encoding](https://www.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WindowsBestFit/bestfit1258.txt). |
+| `bbf_unicode_cp874_ci_as` | Case insensitive collation for the [Windows-874 single-byte character encoding](https://www.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WindowsBestFit/bestfit874.txt). |
+| `sql_latin1_general_cp1250_ci_as` | The same as `bbf_unicode_cp1250_ci_as`. |
+| `sql_latin1_general_cp1251_ci_as` | The same as `bbf_unicode_cp1251_ci_as`. |
+| `sql_latin1_general_cp1_ci_as` | The same as `bbf_unicode_cp1_ci_as`. |
+| `sql_latin1_general_cp1253_ci_as` | The same as `bbf_unicode_cp1253_ci_as`. |
+| `sql_latin1_general_cp1254_ci_as` | The same as `bbf_unicode_cp1254_ci_as`. |
+| `sql_latin1_general_cp1255_ci_as` | The same as `bbf_unicode_cp1255_ci_as`. |
+| `sql_latin1_general_cp1256_ci_as` | The same as `bbf_unicode_cp1256_ci_as`. |
+| `sql_latin1_general_cp1257_ci_as` | The same as `bbf_unicode_cp1257_ci_as`. |
+| `sql_latin1_general_cp1258_ci_as` | The same as `bbf_unicode_cp1258_ci_as`. |
+| `sql_latin1_general_cp874_ci_as` | The same as `bbf_unicode_cp874_ci_as`. |
+| `latin1_general_ci_as` | The same as `sql_latin1_general_cp1_ci_as`. |
+| `latin1_general_bin2` | Binary collation for the [Windows-1252 single-byte character encoding](https://www.unicode.org/Public/MAPPINGS/VENDORS/MICSFT/WindowsBestFit/bestfit1252.txt). |
+| `latin1_general_100_bin2` | The same as `latin1_general_bin2`. |
```

Comment:
the bin2 collations are cs_as, so these are currently not permitted as the server_collation_name